requirements-dev.txt

```diff
@@ -32,7 +32,7 @@ pytest
 scipy
 selenium<4.27.0
 setuptools_scm
-sphinx==7.3.7
+sphinx
```

Comment:
@hansthen you added this pin but it seems that things are working without it. Any reason to keep it that I may be missing?

Reply:
Yes, the problem seems to be only with sphinx 8.2.1.